lib/rack/prx_auth.rb

```diff
@@ -1,34 +1,48 @@
 require 'json/jwt'
 require 'rack/prx_auth/version'
-require 'rack/prx_auth/public_key'
+require 'rack/prx_auth/certificate'
 require 'rack/prx_auth/token_data'
 require 'rack/prx_auth/railtie' if defined?(Rails)
 
 module Rack
   class PrxAuth
-    attr_reader :public_key
+    INVALID_TOKEN = [
+      401, {'Content-Type' => 'application/json'},
+      [{status: 401, error: 'Invalid JSON Web Token'}.to_json]
+    ]
+
+    DEFAULT_ISS = 'id.prx.org'
+
+    attr_reader :public_key, :issuer
```

Comment:
Looks like the public_key instance variable doesn't exist anymore so we don't need an attr_reader for it.